simplipy/system/__init__.py

```diff
@@ -342,6 +345,28 @@ async def _get_system_info(self) -> None:
             location_info["system"]["alarmState"]
         )
 
+        cameras_list = location_info["system"]["cameras"]
+        for camera in cameras_list:
+            entity_type = (
+                EntityTypes.doorbell
+                if camera["model"] == "SS002"
```

Comment:
This shouldn't be hardcoded like this. Perhaps subclass the `Camera` class into a `Doorbell` class and put this value there?

Reply:
I thought of doing that, but a camera and a doorbell are essentially the same thing.. I can add the `SS002` constant to `camera.py` though, it may make it cleaner.